main.go

```diff
@@ -232,13 +233,23 @@ func createDefaultOperatorConfig(cfg *rest.Config) error {
 	if err != nil {
 		return fmt.Errorf("Couldn't create client: %v", err)
 	}
+
+	// Check if we only have one node
```

Comment:
Actually this commit message doesn't align with a code below

Reply:
nit: i think reading L237 is pretty clear, you can drop this comment if you like.

Reply:
done

Reply:
> done

done and not removed ? :)

Reply:
sorry about that